arcgis-ios-sdk-samples/Content Display Logic/Views/GroupLayersSectionView.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+final class GroupLayersSectionView: UITableViewHeaderFooterView {
```

Comment:
Since this view is specific to this sample, please move it into the directory for the sample. `Content Display Logic` is used for views (among other things) that are reusable and/or not specific to any one sample.